plugins/cube_inverter.js

```diff
@@ -0,0 +1,174 @@
+        const icon = "swap_horiz"
+        const author = "SirJain"
+
+        // Used for about dialog
+        const links = {
+                website: "https://sirjain0.github.io/",
+                twitter: "https://twitter.com/SirJain2",
+                discord: "https://discord.gg/wM4CKTbFVN"
+        }
+
+        // Registers plugin data
+        Plugin.register(id, {
+                title: name,
+                icon,
+                author,
+                description: "Adds a button that inverts selected cube sizes.",
+                about: "This plugin adds a button that inverts the size values of each selected cube.\n## How to use\nTo use this plugin, go to the size values of the cube (located in edit mode) and click the `Invert Cubes` button located next to the Inflation value. The button is also keybinded with a default of `Shift + I`. You can add or remove the tool using the `Customize Toolbar` option.\n\nPlease report any bugs or suggestions you may have.",
+                version: "1.0.0",
+                min_version: "4.2.0",
+                variant: "both",
+                oninstall: () => showAbout(true),
+                onload() {
+                        addAbout()
+
+                        cubeInverterAction = new Action("cube_inverter_action", {
+                                name: "Invert Cubes",
+                                icon: icon,
+                                description: "Invert the values of all axes of selected cubes.",
+                                keybind: new Keybind({key: 'i', shift: true}),
+                                condition: () => Cube.selected.length,
+                                click: () => invertCubes()
+                        })
+
+                        Toolbars.element_size.add(cubeInverterAction)
+                },
+                onunload() {
+                        aboutAction.delete()
+                        cubeInverterAction.delete()
+                        MenuBar.removeAction(`help.about_plugins.about_${id}`)
+                        Blockbench.showQuickMessage("Uninstalled Cube Inverter plugin")
+                }
+        })
+
+        function invertCubes() {
+                Undo.initEdit({elements: Cube.selected, outliner: true});
+
+                for (const cube of Cube.selected) {
+
+                        // Handles size dimensions
+                        [cube.from, cube.to] = [cube.to, cube.from]
+
+                        // UV handling
+                        cube.faces.north.uv = cube.faces.south.uv
+                        cube.faces.south.uv = cube.faces.north.uv
+                        cube.faces.north.texture = cube.faces.south.texture 
+                        cube.faces.south.texture = cube.faces.north.texture
+                        cube.faces.north.cullface = cube.faces.south.cullface 
+                        cube.faces.south.cullface = cube.faces.north.cullface
+                        cube.faces.north.rotation += 180
```

Comment:
You might want to make sure that the rotation stays within the permitted values 0, 90, 180, 270.
Modulo solution: `rotation = (rotation + 180) % 360`